policies/design-process.md

```diff
@@ -0,0 +1,154 @@
+workshop to collectively identify the candidate solutions. In this case it will
+typically still be necessary for the design team to document the solution
+options outside of the workshop.
+
+The design team may choose to make recommendations about the preferred solution
+option.
+
+### Select solution
+
+Once the candidate solutions have been identified and documented a solution
+must be selected by the OTC. There should be an opportunity for the OpenSSL
+community or non-OTC members to provide input into the decision. This could
+either be in the form of comments on a draft Pull Request describing the
+candidate solutions, or it could be in the form of an open invite for community
+members to join the workshop where the solution is selected.
+
+Optionally the OTC could select no solution and instead direct the design team
+to iterate over the candidate solutions again and either make refinements or
+identify additional options.
+
+### Refine and document
+
+In this step the design team provide further detail on the selected solution
+option. There should be sufficient detail to enable the implementation work to
+progress and to understand the trajectory of the solution into the future. Some
+details may be omitted until a later stage of the development project if
+appropriate and they are not needed until later. The assumptions of a solution
+should be clearly listed along with any omissions.
+
+The design document should be available for public comment in the form of a
+pull request.
+
+### Agree solution
+
+In this step the OTC agree the solution as it is described in the design
+document as produced by the design team. Once agreed the implementation may
+proceed and the design document should be merged into the main source repository.
+
+The OTC may choose not to agree a solution. In this case the process should
+iterate back to the "Refine and document" step to incorporate feedback from the
+OTC. In some cases more significant rework may be required, in which case the
+OTC may choose to step back to any previous step in this design process.
+
+In some cases the OTC may agree a solution which has certain details
+deliberately omitted in order to defer work on those details until a later stage
+of the implementation project. In such cases this process iterates back to the
+"Refine and document" stage to fill in those details when they are eventually
+required, and the subsequent amended design document must be agreed again by the
+OTC.
+
+Transparency and Community Involvement
+--------------------------------------
+
+It should be possible for community members to see the progress of designs
+through the process and to pass comment on the proposed design prior to its
+eventual agreement by OTC. As a minimum:
+
+* The draft design document containing the candidate solutions should be
+available as a pull request for public comment for at least one week prior to
+the selection of the solution. As an alternative, community members may be
```

Comment:
IMO the one week for the PR before the selection should be always kept. So perhaps: `In addition to that, community members may...`